core/store/src/trie/mod.rs

```diff
@@ -562,11 +561,11 @@ impl WrappedTrieChanges {
         store_update: &mut StoreUpdate,
     ) -> Result<(), Box<dyn std::error::Error>> {
         store_update.trie = Some(self.trie.clone());
-        for (key, raw_changes) in
+        for (key, change_with_trie_key) in
             std::mem::replace(&mut self.state_changes, Default::default()).into_iter()
```

Comment:
While you are at it, and since we have recently updated Rust compiler, update this `std::mem::replace` with [`std::mem::take`](https://doc.rust-lang.org/stable/std/mem/fn.take.html)